config/generate-key/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use libra_crypto::{ed25519::Ed25519PrivateKey, Uniform};
+use rand::{rngs::OsRng, Rng, SeedableRng};
+use std::{
+    fs::{self, File},
+    io::Write,
+    path::Path,
+};
+
+pub fn generate_key() -> Ed25519PrivateKey {
+    let mut seed_rng = OsRng;
+    let mut rng = rand::rngs::StdRng::from_seed(seed_rng.gen());
+    Ed25519PrivateKey::generate(&mut rng)
```

Comment:
never seed an RNG yourself: use [ThreadRng](https://docs.rs/rand/0.7.3/rand/rngs/struct.ThreadRng.html) instead

Reply:
This is just a code mod generate-keypair -> generate-key

Your argument makes sense though... but out of scope for this PR.